src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -161,6 +161,45 @@ export default async function UserBreaches() {
     return breachRowsHTML;
   }
 
+  // Fetch list of subscriptions.
+  const bearerToken = session?.user.subscriber?.fxa_access_token;
+  if (!bearerToken) {
+    console.error("User has no bearer token");
+  }
+  const result = await fetch(
+    `${process.env.OAUTH_API_URI}/oauth/mozilla-subscriptions/customer/billing-and-subscriptions`,
+    {
+      headers: {
+        "Content-Type": "application/json",
+        authorization: `Bearer ${bearerToken}`,
+      },
+    }
+  );
+  let subscriptions;
+  if (result.ok) {
+    subscriptions = await result.json();
+  } else {
+    console.error("Could not refresh subscriptions:", result);
+  }
```

Comment:
Might it make sense to turn this into a function that can also be called from elsewhere? Also, I'd really appreciate type definitions for the return type, even if they only cover what we actually use for now :) e.g.

```ts
type FxaSubscriptionResponse = {
  subscriptions: Array<{
    product_id: string;
    plan_id: string;
    status: "active";
  }>;
};
```